AppCenterDataStorage/AppCenterDataStorage/Internal/Ingestion/MSStorageIngestion.m

```diff
@@ -0,0 +1,62 @@
+#import "MSAppCenter.h"
+#import "MSAppCenterInternal.h"
+#import "MSStorageIngestion.h"
+#import "MSHttpIngestionPrivate.h"
```

Comment:
Sort imports alphabetically. You can solve most of formatting issues by run `clang-format-changed-files.sh` script